specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-05-01-preview/examples/RunImageTemplate.json

```diff
@@ -2,15 +2,15 @@
   "parameters": {
     "subscriptionId": "{subscription-id}",
     "resourceGroupName": "myResourceGroup",
-    "imageTemplateName": "myImageTemplate",
+    "imageTemplateName":"myImageTemplate",
```

Comment:
Previous formatting was correct in this file

Reply:
fixed this and the others